lib/datarockets_style/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module DatarocketsStyle
-  VERSION = "1.5.0"
```

Comment:
It should be part of https://github.com/datarockets/ruby-style/blob/master/RELEASING.md